app-dicts/myspell-en/myspell-en-20240701.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit myspell-r2
+
+DESCRIPTION="English dictionaries for myspell/hunspell"
+HOMEPAGE="
+	https://extensions.libreoffice.org/extensions/english-dictionaries
+	https://github.com/marcoagpinto/aoo-mozilla-en-dict
+	https://proofingtoolgui.org
+"
+SRC_URI="https://extensions.libreoffice.org/assets/downloads/41/1719812763/dict-en-20240701_lo.oxt"
```

Comment:
Why is this not using ${PV} anymore?

Reply:
Option 1. Copy and paste the link.
Option 2. Copy and paste the link. Replace _the part_ of what changed since last month by `$PV`.

Which one is simpler?
Which one is safer?
Is option 2 more illustrative?